libmcount/mcount.h

```diff
@@ -114,7 +114,7 @@ struct mcount_shmem {
 };
 
 /* first 4 byte saves the actual size of the argbuf */
-#define ARGBUF_SIZE  1024
+#define ARGBUF_SIZE  16384
```

Comment:
I don't think this is needed since it's not for passing argpsec.  It is used to save actual contents of arguments and return value.

Reply:
I see. I just limited the maximum length to 64 in 29ce50c. Thanks!